native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -186,9 +186,7 @@ static char *normalize_hostname(apr_pool_t *p, const char *hostname)
     return ret;
 }
 
-/**
- * Normalize the worker name
- */
```

Comment:
I agree the format should be one and we should avoid mixing:
/*
 * Comment
 */
and
/* Comment */

But I would prefer keeping the first one, except you have a good justification.

Reply:
Looking at it again, I would suggest leaving this as is and creating a new issue to unify the styling for the whole code base (and not only mod_proxy_cluster.c). WDYT?

Reply:
Yes please ;-)

Reply:
Ok, I opened #158